Formula/sshguard.rb

```diff
@@ -16,12 +16,22 @@ class Sshguard < Formula
   depends_on "autoconf" => :build
```

Comment:
Does it actually need `automake` and `autoconf`? It seems that the `configure` script is already present.